### PR TITLE
fix the error of invalid min gas price in evm tx

### PR DESCRIPTION
**Pull request by AaronZgl**
- Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/okex/okexchain/blob/develop/CONTRIBUTING.md#pr-targeting))

- [ ] Linked to github-issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote tests
- [ ] Updated relevant documentation (`docs/`)
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Reviewed `Files changed` in the github PR explorer

______

For Admin Use:
- Added appropriate labels to PR (ex. wip, ready-for-review, docs)
- Reviewers Assigned
- Squashed all commits, uses message "Merge PR #XYZ: [title]" ([coding standards](https://github.com/okex/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

**Review by zhongqiuwood — COMMENTED** [review]
ut requried

File: app/ante/eth.go [hunk 1]
```diff
@@ -103,7 +104,7 @@ func (emfd EthMempoolFeeDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simula
 	// fee = gas price * gas limit
 	fee := sdk.NewDecCoin(evmDenom, sdk.NewIntFromBigInt(msgEthTx.Fee()))
 
-	minGasPrices := ctx.MinGasPrices()
+	minGasPrices := ctx.MinGasPrices().MulDec(sdk.NewDecFromBigInt(math.BigPow(10, sdk.Precision)))
```
Comment (zhongqiuwood) [inline app/ante/eth.go]: magic number is not allowed

**Comment by AaronZgl** [comment]
> ut requried

Done!

**Review by zhongqiuwood — DISMISSED** [review]

File: app/ante/eth.go (hunk 1 above)
Comment (AaronZgl) [reply app/ante/eth.go]: fixed it!

File: app/ante/eth.go [hunk 2]
```diff
@@ -103,7 +104,8 @@ func (emfd EthMempoolFeeDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simula
 	// fee = gas price * gas limit
 	fee := sdk.NewDecCoin(evmDenom, sdk.NewIntFromBigInt(msgEthTx.Fee()))
 
-	minGasPrices := ctx.MinGasPrices()
+	coinUnit := sdk.NewDecFromBigInt(math.BigPow(10, sdk.Precision))
```
Comment (KamiD) [inline app/ante/eth.go:107]: magic number is not allowed, what's the exactly means of 10

**Review by KamiD — CHANGES_REQUESTED** [review]

**Pull request closed without merging**